x-pack/solutions/security/plugins/security_solution/public/common/hooks/timeline/use_investigate_in_timeline.ts

```diff
@@ -124,19 +129,35 @@ export const useInvestigateInTimeline = () => {
         // Only show detection alerts
         // (This is required so the timeline event count matches the prevalence count)
         if (!keepDataView) {
-          dispatch(
-            sourcererActions.setSelectedDataView({
-              id: SourcererScopeName.timeline,
-              selectedDataViewId: defaultDataView.id,
-              selectedPatterns: [signalIndexName || ''],
-            })
-          );
+          if (newDataViewPickerEnabled) {
+            setSelectedDataView({
+              scope: [SourcererScopeName.timeline],
+              id: defaultDataView.id,
+              fallbackPatterns: [signalIndexName || ''],
```

Comment:
I understand it for current compatibility, but in the long run, we _shouldn't_ need fallbackPatterns anymore?

Reply:
yeah and since this is for timelines specifically maybe I should include that in the name somehow?